libraries/lib-effects/EffectOutputTracks.cpp

```diff
@@ -38,7 +39,9 @@ EffectOutputTracks::EffectOutputTracks(
    auto count = 0;
    auto reportProgress = [&](double progressFraction) {
       const auto overallProgress = (count + progressFraction) / numTracks;
-      progress->Poll(overallProgress * 1000, 1000);
+      const auto result = progress->Poll(overallProgress * 1000, 1000);
```

Comment:
Done.

I recall there was a difference between Stop and Cancel for import and export.  I didn't look to see if that was conserved after all the refactoring in this release.

It was possible to Stop with a partially exported file instead of aborting the whole thing and deleting the file.